### PR TITLE
Update Copilot icon

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request:
- Updates the Copilot icon
- Applies /get-started edits that I came across in reviewing the AI and Copilot docs

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/get-started/library.md [hunk 1]
```diff
@@ -226,7 +226,7 @@ To rename a pinned search:
 To remove a search from the pinned search List:
 
 1. Hover over the search, then click the three-dot kebab menu icon to the right of the name.
-1. Click **Unpin**. 
+1. Click **Unpin**.
```
Comment (kimsauce) [inline docs/get-started/library.md:229]:
```suggestion
1. Click **Unpin**. 
```

File: docs/get-started/library.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ When your content is moved, you’ll need to click the new folder location to vi
 To add a folder to the Library:
 
 1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> At the top of the screen, select the **Library** (folder) icon. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Library**. You can also click the **Go To...** menu at the top of the screen and select **Library**. Kanso-->
-1. Click **Add New > New Folder**. 
+1. Click **Add New > New Folder**.
```
Comment (kimsauce) [inline docs/get-started/library.md:82]:
```suggestion
1. Click **Add New > New Folder**. 
```

File: docs/get-started/library.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+sidebar_label: Managing Your Library
 description: Managing your Library in Sumo Logic.
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
 You can save, share, and manage Searches, Dashboards, Apps, and other Sumo Logic content in the Library. The Library also allows you to launch searches and Dashboards with a single click—speeding up access to the searches you find yourself running consistently. Additionally, you can use the content that others in your organization have already developed to continually discover new insights in your data.
 
-Open the **Library** page to expose additional search options and to organize and manage Library contents. 
+Open the **Library** page to expose additional search options and to organize and manage Library contents.
```
Comment (kimsauce) [inline docs/get-started/library.md:12]:
```suggestion
Open the **Library** page to expose additional search options and to organize and manage Library contents. 
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@kimsauce, the old icon still appears in the upper left of many screenshots in the Copilot article. You'll want to update those.

source-category

**Comment by kimsauce** [comment]
> @kimsauce, the old icon still appears in the upper left of many screenshots in the Copilot article. You'll want to update those.
> 
> source-category

Oh, good call! I completely missed that. I'll fix and resubmit.

**Comment by kimsauce** [comment]
Thanks again for catching those images, @jpipkin1 ! I've fixed them all except for the video and gif. Since the product is so in flux, those will likely be updated anyway.

**Review by jpipkin1 — APPROVED** [review]
Looks great, @kimsauce!

**Pull request merged**